### ISSUE TITLE
Switch to use coala-utils.Question

### ISSUE THREAD

**Issue by yzgyyang** [coala/coala-quickstart#261]
IMO we should switch to use components in `coala-utils` like this one:

https://github.com/coala/coala-quickstart/blob/master/coala_quickstart/generation/SettingsFilling.py#L215

Use this instead:
https://gitlab.com/coala/coala-utils/blob/master/coala_utils/Question.py

(Note that it's currently blocked by https://gitlab.com/coala/coala-utils/issues/63)

**Comment by jayvdb** [comment]
To do this, the repo will need to install coala-utils, coala & coala-bears from git, not pypi

Something like https://github.com/coala/coala-bears/commit/6906e5465123ecd46e41b603b725bb4d188a23c6

However this will create a version conflict with coala-utils, as coala doesnt use coala-utils master.  That needs to be done first.  And that depends on https://github.com/di/pytest-reqs/issues/16 :P

**Comment by jayvdb** [comment]
Did a bunch of PRs to get that moving https://github.com/di/pytest-reqs/pulls

**Comment by jayvdb** [comment]
https://pypi.org/project/argparse-prompt/ looks very interesting.

It looks like that might be very suitable for quickstart, allowing lots of options to be exposed to the CLI, or be deferred to the user as prompts.

It may not work for all of the prompts currently in quickstart, but even moving a few of the custom prompts to become argparse-prompt would be a big win for usability.
